keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -1210,7 +1211,8 @@ class SqlArtifactRepository(
     deliveryConfig: DeliveryConfig,
     environmentName: String,
     artifactReference: String,
-    version: String
+    version: String,
+    verifications: List<ArtifactVersionEnvironmentVerificationSummary>
```

Comment:
Why are you passing this in here only to just return it? where will this actually get calculated?

Reply:
I'm going to change this to the copy pattern that @luispollo mentioned in [his comment](https://github.com/spinnaker/keel/pull/1777#discussion_r568965217).